frontend/components/ChoiceControlledContent/index.js

```diff
@@ -0,0 +1,32 @@
+import "./index.css";
+
+class ChoiceControlledContent {
+  constructor(node) {
+    this.node = node;
+    this.controlledByNode = document.querySelector(
+      this.node.getAttribute("data-controlled-by")
+    );
+    this.handleChangeEvent = this.handleChangeEvent.bind(this);
+    this.addControllerListener = this.addControllerListener.bind(this);
+
+    this.addControllerListener();
+  }
+
+  addControllerListener() {
```

Comment:
Should this be `addedControlledListener` I'm confused by the term `Controller` in this context.

Reply:
`addControllerListener` refers to the `controlled_by` parameter.